### PR TITLE
docs: split custom exception handler and scaling pages

### LINKED ISSUE #825: Custom Exception Handler and  application level docs are intermixed
https://robyn.tech/documentation/api_reference/exceptions#scaling-the-application

Make two separate pages

### PR DESCRIPTION
**Pull request by VishnuSanal**
**Description**

This PR fixes #825

### CODE REVIEW

**Comment by sansyrox** [comment]
Hey @VishnuSanal 👋 

Could you please add the links here as well (https://github.com/sparckles/Robyn/blob/3644f21ce9984758fdf9b0dc9e03ea7094118fe1/docs_src/src/components/documentation/ApiDocs.jsx#L4)

File: docs_src/src/pages/documentation/api_reference/exceptions.mdx [hunk 1]
```diff
@@ -32,7 +27,7 @@ Batman scaled his application across multiple cores for better performance. He u
 
 ## What's next?
 
-Now, Batman wanted to extend Robyn. Robyn told him about the plugins.
+Now, Batman wanted to scaled his application across multiple cores. Robyn led him to Scaling.
 
-- [Plugins](/documentation/plugins)
+- [Scaling](/documentation/scaling)
```
Comment (sansyrox) [inline docs_src/src/pages/documentation/api_reference/exceptions.mdx:32]: @VishnuSanal , this should be `documentation/api_reference/scaling` here

File: docs_src/src/pages/documentation/api_reference/scaling.mdx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  </Col>
+</Row>
+
+---
+
+## What's next?
+
+Now, Batman wanted to extend Robyn. Robyn told him about the plugins.
+
+- [Plugins](/documentation/plugins)
```
Comment (sansyrox) [inline docs_src/src/pages/documentation/api_reference/scaling.mdx:28]: could probably make another issue for this but now that I look at it, the navigation does not match the layout, we should probably talk about the plugins at the end.

File: docs_src/src/pages/documentation/api_reference/exceptions.mdx [hunk 3]
```diff
@@ -32,7 +27,7 @@ Batman scaled his application across multiple cores for better performance. He u
 
 ## What's next?
 
-Now, Batman wanted to extend Robyn. Robyn told him about the plugins.
+Now, Batman wanted to scaled his application across multiple cores. Robyn led him to Scaling.
```
Comment (sansyrox) [inline docs_src/src/pages/documentation/api_reference/exceptions.mdx:30]:
```suggestion
Now, Batman wanted to scale his application across multiple cores. Robyn led him to Scaling.
```

**Comment by sansyrox** [comment]
Hey @VishnuSanal , just some inline comments :D

File: docs_src/src/pages/documentation/api_reference/scaling.mdx (hunk 2 above)
Comment (VishnuSanal) [reply docs_src/src/pages/documentation/api_reference/scaling.mdx:28]: moved to #844

File: docs_src/src/pages/documentation/api_reference/exceptions.mdx [hunk 4]
```diff
@@ -1,13 +1,8 @@
 ## Custom Exception Handler
```
Comment (sansyrox) [inline docs_src/src/pages/documentation/api_reference/exceptions.mdx:1]:
```suggestion
## Custom Exception Handler

Batman learned how to create custom error handlers for different exception types in his application. He wrote the following code to handle exceptions and return a custom error response:
```

File: docs_src/src/pages/documentation/api_reference/exceptions.mdx [hunk 5]
```diff
@@ -1,13 +1,8 @@
...
 
-Batman learned how to create custom error handlers for different exception types in his application. He wrote the following code to handle exceptions and return a custom error response:
-
-
-## Scaling the Application
-
 <Row>
 <Col>
-Batman scaled his application across multiple cores for better performance. He used the following command:
+Batman learned how to create custom error handlers for different exception types in his application. He wrote the following code to handle exceptions and return a custom error response:
```
Comment (sansyrox) [inline docs_src/src/pages/documentation/api_reference/exceptions.mdx:5]: @VishnuSanal , just one final change - could you 
```suggestion
```

**Review by sansyrox — APPROVED** [review]
LGTM! Great work @VishnuSanal 😄

**Pull request merged**